register.lic

```diff
@@ -22,13 +22,14 @@ class Register
     container = get_settings.crafting_container
 
     stow_hands
-    if get_item('deed register', container)
-      bput('turn my register to contents', 'You flip your deed register to the contents page.', 'But a deed register is already at the table of contents!')
-      search(args.query)
-    else
-      echo "Can\'t find your deed register."
-      exit
+    unless get_item('deed register', container)
+      unless bput('get my deed register', /You get/, /What were you referring to/) == 'You get'
```

Comment:
💡 This my read a little clearer as `if bput(...) != 'You get'`

Reply:
True.  I was using an if at first but switched to unless later.  I can always change it if it would be better for the overall style we're aiming for.